packages/core/stories/all.stories.ts

```diff
@@ -144,10 +144,9 @@ if (process.env.NODE_ENV !== "production") {
 		storyUtils.addRadioButtonEventListeners(container);
 
 		const getNewRow = () => {
-			const row = document.createElement("div");
-			row.className = "bx--row";
-
-			return row;
```

Comment:
wasn't working?

Reply:
it's just to be more coherent with previous code in the same file a few lines above